runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -522,7 +523,11 @@ private void contract()
 		giveClientFocus();
 		SwingUtil.revalidateMinimumSize(frame);
 
-		if (wasMinimumWidth)
+		if ((frame.getExtendedState() & Frame.MAXIMIZED_BOTH) == Frame.MAXIMIZED_BOTH)
```

Comment:
This might be more clear if you put all of this in `if ((frame.getExtendedState() & MAXIMIZED_BOTH) != MAXIMIZED_BOTH)` instead